collections/test.ts

```diff
@@ -0,0 +1,161 @@
+import { Mongo } from 'meteor/mongo';
```

Comment:
This code should 100% not be in /collections.  I would prefer if you just kept this data on the example database rather than in code form, but if it has to go somewhere, it should be in /tests.